### PR TITLE
Added more description to the spacing documentation

### LINKED ISSUE #5601: [docs] Spacing documentation is unclear
This [page](https://github.com/quasarframework/quasar/blob/dev/docs/src/pages/style/spacing.md#syntax) does not really explain what each of those options do. I'm at a loss to understand how to apply that.

### PR DESCRIPTION
**Pull request by InfoSec812**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [x] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] It's submitted to the `dev` branch and _not_ the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [x] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

Resolves #5601

### CODE REVIEW

**Comment by pdanpdan** [comment]
This looks very nice, thank you.

**Review by pdanpdan — CHANGES_REQUESTED** [review]
Please remove `(ONLY for specific margins: q-ml-*, q-mr-*, q-mx-*)` from all rows and check to only show `auto` for all `q-p...` and `q-ml-...`, `q-mr-...` and `q-mx-...`

**Comment by InfoSec812** [comment]
> Please remove `(ONLY for specific margins: q-ml-*, q-mr-*, q-mx-*)` from all rows and check to only show `auto` for all `q-p...` and `q-ml-...`, `q-mr-...` and `q-mx-...`

@pdanpdan I think that this should be corrected

File: docs/src/pages/style/spacing.md [hunk 1]
```diff
@@ -56,98 +56,91 @@ Examples: `q-pa-xs-md q-pa-sm-sm q-px-md-lg q-py-md-md`
...
 | `q-` | `m` (margin) | `l` (left) | `xl` (extra large) | `q-ml-xl` |
 | `q-` | `m` (margin) | `a` (all) | `none` () | `q-ma-none` |
-| `q-` | `m` (margin) | `a` (all) | `auto` (ONLY for specific margins: q-ml-*, q-mr-*, q-mx-*) | `q-ma-auto` |
 | `q-` | `m` (margin) | `a` (all) | `xs` (extra small) | `q-ma-xs` |
 | `q-` | `m` (margin) | `a` (all) | `sm` (small) | `q-ma-sm` |
 | `q-` | `m` (margin) | `a` (all) | `md` (medium) | `q-ma-md` |
 | `q-` | `m` (margin) | `a` (all) | `lg` (large) | `q-ma-lg` |
 | `q-` | `m` (margin) | `a` (all) | `xl` (extra large) | `q-ma-xl` |
 | `q-` | `m` (margin) | `x` (left & right) | `none` () | `q-mx-none` |
-| `q-` | `m` (margin) | `x` (left & right) | `auto` (ONLY for specific margins: q-ml-*, q-mr-*, q-mx-*) | `q-mx-auto` |
```
Comment (pdanpdan) [inline docs/src/pages/style/spacing.md]: This should stay (without comment)

File: docs/src/pages/style/spacing.md [hunk 2]
```diff
@@ -56,98 +56,91 @@ Examples: `q-pa-xs-md q-pa-sm-sm q-px-md-lg q-py-md-md`
...
 | `q-` | `m` (margin) | `r` (right) | `xl` (extra large) | `q-mr-xl` |
 | `q-` | `m` (margin) | `b` (bottom) | `none` () | `q-mb-none` |
-| `q-` | `m` (margin) | `b` (bottom) | `auto` (ONLY for specific margins: q-ml-*, q-mr-*, q-mx-*) | `q-mb-auto` |
 | `q-` | `m` (margin) | `b` (bottom) | `xs` (extra small) | `q-mb-xs` |
 | `q-` | `m` (margin) | `b` (bottom) | `sm` (small) | `q-mb-sm` |
 | `q-` | `m` (margin) | `b` (bottom) | `md` (medium) | `q-mb-md` |
 | `q-` | `m` (margin) | `b` (bottom) | `lg` (large) | `q-mb-lg` |
 | `q-` | `m` (margin) | `b` (bottom) | `xl` (extra large) | `q-mb-xl` |
 | `q-` | `m` (margin) | `l` (left) | `none` () | `q-ml-none` |
-| `q-` | `m` (margin) | `l` (left) | `auto` (ONLY for specific margins: q-ml-*, q-mr-*, q-mx-*) | `q-ml-auto` |
```
Comment (pdanpdan) [inline docs/src/pages/style/spacing.md]: This should stay (without comment)

File: docs/src/pages/style/spacing.md [hunk 3]
```diff
@@ -56,98 +56,91 @@ Examples: `q-pa-xs-md q-pa-sm-sm q-px-md-lg q-py-md-md`
...
 | `q-` | `p` (padding) | `y` (top & bottom) | `xl` (extra large) | `q-py-xl` |
 | `q-` | `m` (margin) | `t` (top) | `none` () | `q-mt-none` |
-| `q-` | `m` (margin) | `t` (top) | `auto` (ONLY for specific margins: q-ml-*, q-mr-*, q-mx-*) | `q-mt-auto` |
 | `q-` | `m` (margin) | `t` (top) | `xs` (extra small) | `q-mt-xs` |
 | `q-` | `m` (margin) | `t` (top) | `sm` (small) | `q-mt-sm` |
 | `q-` | `m` (margin) | `t` (top) | `md` (medium) | `q-mt-md` |
 | `q-` | `m` (margin) | `t` (top) | `lg` (large) | `q-mt-lg` |
 | `q-` | `m` (margin) | `t` (top) | `xl` (extra large) | `q-mt-xl` |
 | `q-` | `m` (margin) | `r` (right) | `none` () | `q-mr-none` |
-| `q-` | `m` (margin) | `r` (right) | `auto` (ONLY for specific margins: q-ml-*, q-mr-*, q-mx-*) | `q-mr-auto` |
```
Comment (pdanpdan) [inline docs/src/pages/style/spacing.md]: This should stay (without comment)

**Review by pdanpdan — CHANGES_REQUESTED** [review]

**Comment by rstoenescu** [comment]
Thank you for contributing!

**Pull request merged**